docs/source/tutorial.rst

```diff
@@ -51,18 +39,18 @@ There are three ways to complete this tutorial:
    If you have Docker installed, you can run the tutorial notebook from a pre-configured
    docker image with:
 
-   .. code:: bash
+   .. code-block:: bash
 
-      docker run -it --rm -p 8888:8888 -p 3000:3000 bentoml/quickstart:latest
+      » docker run -it --rm -p 8888:8888 -p 3000:3000 -p 3001:3001 bentoml/quickstart:latest
```

Comment:
can't we use the docker expose ports thing since we use `EXPOSE`?

Reply:
EXPOSE is just documentation. We still need to specify the ports.